### ISSUE TITLE
Third graph print option: Math markdown

### ISSUE THREAD

**Issue by jacobmenick** [Theano/Theano#2784]
It would be great if you had a function that takes a graph as input and produces math markdown. Seems it wouldn't be too hard to parse the output of, say ```pp(f.maker.fgraph.outputs[0])``` manually.

**Comment by nouiz** [comment]
I think that we should navigate the graph or allow pp to print 2 different
format. Parsing the output of pp is probably not the best way I think.

Do you want to try it? All the code for pp is in the file
theano/printing.py.
